src/DynamoCoreWpf/UI/Themes/Modern/Connectors.xaml

```diff
@@ -92,16 +207,38 @@
                         </PathGeometry.Figures>
                     </PathGeometry>
                 </Path.Data>
+                <Path.Style>
+                    <Style TargetType="Path">
+                        <Setter Property="Visibility" Value="Hidden"/>
+                        <Style.Triggers>
+                            <!--show thick wire-->
+                            <MultiDataTrigger>
+                                <MultiDataTrigger.Conditions>
+                                    <Condition Binding="{Binding Path=ConnectorAnchorViewModelExists, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}" Value="True"/>
+                                    <Condition Binding="{Binding Path=PlineVisibility, Mode=OneWay, UpdateSourceTrigger=PropertyChanged}" Value="True"/>
+                                </MultiDataTrigger.Conditions>
+                                <MultiDataTrigger.Setters>
+                                    <Setter Property="Visibility" Value="Visible"/>
+                                </MultiDataTrigger.Setters>
+                            </MultiDataTrigger>
+                        </Style.Triggers>
+                    </Style>
+                </Path.Style>
             </Path>
 
+            <ContentControl Canvas.ZIndex="2"  Content="{Binding ConnectorAnchorViewModel}"
```

Comment:
Comments on ZIndex usage here please, thanks!

Reply:
Turns out this ZIndex was doing nothing, so I removed it.